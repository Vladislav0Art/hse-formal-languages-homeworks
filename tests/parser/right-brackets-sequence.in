start=$S
$S -> '(' $S ')' | NULL;