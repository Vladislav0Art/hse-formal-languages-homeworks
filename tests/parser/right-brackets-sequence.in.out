Grammar(terminals={')', '('}, non_terminals={'S'}, start='S', rules=[Bind(name='S', expr=Alt(values=[Combination(values=[Terminal(value='('), NonTerminal(value='S'), Terminal(value=')')]), Null()]))])
